GAD 3 - Dynamisches Array/tests/dynamicarray/DynamicArrayTester.java

```diff
@@ -6,11 +6,7 @@
 
 import java.util.Random;
 
-/**
- * Tests included from Zulip: https://zulip.in.tum.de/#narrow/stream/419-GAD-E03-Dynamisches.20Array/topic/Tests/near/177724
- * @Author Aamin Dev
```

Comment:
According to this source the A in @Author should be small.
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@author
Also since this is being reconsidered, it's better to leave it at just "Aamin"

Reply:
I re-added the source. Is it okay this way now?

Reply:
Perfect